YIF.Core.Service/Concrete/Services/IoEAdminService.cs

```diff
@@ -353,5 +353,27 @@ public async Task<ResponseApiModel<IEnumerable<LectorResponseApiModel>>> GetIoEL
                 Success = true
             };
         }
+
+        public async Task<ResponseApiModel<DescriptionResponseApiModel>> DeleteIoELector(string lectorId, string userId)
+        {
+            var result = new ResponseApiModel<DescriptionResponseApiModel>();
+            string ioEId = (await _institutionOfEducationAdminRepository.GetByUserId(userId)).InstitutionOfEducationId;
+            var lector = await _lectorRepository.GetLectorByLectorIdAndIoEId(lectorId, ioEId);
+            if (lector == null)
+            {
+                throw new NotFoundException($"{_resourceManager.GetString("IoELectorWithSuchIdNotFound")}: {lectorId}");
+            }
+            else if( lector.IsDeleted == true)
+            {
+                throw new BadRequestException(_resourceManager.GetString("IoELectorWasAlreadyDeleted"));
+            }
+            var searchUser = await _userManager.FindByIdAsync(lector.UserId);
+
+            await _lectorRepository.Delete(lector.Id);
+            await _userManager.RemoveFromRoleAsync(searchUser, ProjectRoles.Lector);
+            await _userRepository.Delete(searchUser.Id);
+            
+            return result.Set(new DescriptionResponseApiModel("IoELectorIsDeleted"), true);
```

Comment:
you didn't IoELectorIsDeleted add to resource menager

Reply:
Fixed